account_voucher_tax/tests/common.py

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8 -*-
+
+from openerp.tests import common
+import time
+
+
+class TestTaxCommon(common.TransactionCase):
+
+    def setUp(self):
+        super(TestTaxCommon, self).setUp()
+        self.account_invoice_model = self.registry('account.invoice')
+        self.account_invoice_line_model = self.registry('account.invoice.line')
+        self.acc_bank_stmt_model = self.registry('account.bank.statement')
+        self.acc_bank_stmt_line_model = self.registry(
+            'account.bank.statement.line')
+        self.partner_agrolait_id = self.registry(
+            "ir.model.data").get_object_reference(
```

Comment:
With the new api exists a method called ref, this method is used for get the record of the xml_id.
e.g.
self.env.ref('module.xml_id')